winlink_monitor.py

```diff
@@ -83,15 +84,23 @@ def load_config(args):
     '''
     config_json = json.load(open(args.config, 'r'))
 
-    # Time to wait (seconds) between sending a probe and checking for it. There's then exponential backoff for the retries.
+    # Time to wait (seconds) between sending a probe and checking for it.
+    # There's then exponential backoff for the retries.
     CONFIG['fetch_retry_interval_seconds'] = int(config_json.get("fetch_retry_interval_seconds", 30))
 
-    # Number of times to retry looking for a probe. (There's exponential backoff between them - see interval, above)
+    # Number of times to retry looking for a probe.
+    # There's exponential backoff between them - see interval, above.
     CONFIG['fetch_retries_count'] = int(config_json.get("fetch_retries_count", 3))
 
     # How many runs we look back at for determining health
     CONFIG['health_window_size'] = int(config_json.get("health_window_size", 5))
 
+    # How much health history we should keep.  Anything more than health_window_size
+    # is for human consumption through the web interface to assess historicl health.
+    # Force this to be at least health_window_size.
```

Comment:
Do we want to silently upgrade or warn/error out? 

I'm generally of the "be pedantic and fail ASAP so the user realizes they have to fix it" mindset but I don't feel strongly.